src/job/models/ingestion/updateJobHandler.ts

```diff
@@ -14,7 +14,7 @@ import { CatalogClient } from '../../../httpClients/catalogClient';
 import type { IConfig, IJobHandler, MergeTilesTaskParams } from '../../../common/interfaces';
 import { JobTrackerClient } from '../../../httpClients/jobTrackerClient';
 import { Grid } from '../../../common/interfaces';
-import { SeedMode, SERVICES } from '../../../common/constants';
+import { SERVICES } from '../../../common/constants';
```

Comment:
`SeedMode` constant and equivalent type are no longer used
please remove them

Reply:
They are in use in other places